www/ifarchive-tuid-report

```diff
@@ -87,9 +88,20 @@ $db = dbConnect();
 
 $output = [];
 foreach($tuids as $tuid => $path) {
-    $result = mysqli_execute_query($db, "select url from gamelinks where gameid = ? and url like ?", [$tuid, "%$path"]);
-    $row = mysqli_fetch_row($result);
-    if (!$row) {
+    // normalize the path using URL normalizer. First we make it an absolute URL, normalize it, then we strip the absolute part off
+    $path = substr((new URL\Normalizer("https://ifarchive.org/$path"))->normalize(), strlen("https://ifarchive.org/"));
```

Comment:
I would put `https://ifarchive.org/` in a variable since it's used twice, and explicitly required to be identical for this "get suffix" technique.